Makefile

```
@@ -0,0 +1,185 @@
+.PHONY: help all clean install install-dev install-backend install-frontend \
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)